test/services/oauth2/hooks/index.test.js

```diff
@@ -0,0 +1,88 @@
+const { expect } = require('chai');
+const appPromise = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(appPromise);
+
+const { setIdToken } = require('../../../../src/services/oauth2/hooks/index');
+
+describe.only('oauth2 token test', () => {
+	let app;
+	let server;
+    let createHook;
```

Comment:
I first wrote this into the review message itself, which is easily overlooked, so I will repeat it here again: there are a bunch of linter errors reported by codacy scattered all over these files (most of them seem to be whitespaces).

Reply:
fixed that! :)